src/cloudsc2_nl_loki/README.md

```diff
@@ -0,0 +1,4 @@
+Current implementation provides working idem and scc variants, the latter with 3.5 GFlops/s. scc-hoist currently requires manual edit of 
```

Comment:
Don't think this is needed anymore. Instead maybe update the central README with latest state?